torch/csrc/jit/passes/inline_autodiff_subgraphs.cpp

```diff
@@ -21,7 +21,8 @@ bool canRunWithAutograd(Node* node) {
   }
   return kind != prim::FusionGroup && kind != prim::CudaFusionGroup &&
       kind != prim::TypeCheck && kind != prim::TensorExprGroup &&
-      kind != prim::CudaFusionGuard && (kind.is_aten() || kind.is_prim());
+      kind != prim::RequiresGradCheck && kind != prim::CudaFusionGuard &&
```

Comment:
why did you  add this here ?

Reply:
you are right. it doesn't make too much sense since all RequiresGradCheck should be **outside** :-) or differentiable graphs

Reply:
removed it.